### PR TITLE
Lint code to conform with pep8

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR lints the code so it would conform to pep8 by doing the following:
1. Ran `ooipy/` and `setup.py` through `black`
2. Ran `ooipy/` and `setup.py` through `isort`
3. Ran `ooipy/` through `flake8` and fix as needed.
4. Updated `.pre-commit-config.yaml` for lint test to run with updated tools.

### CODE REVIEW

**Comment by lsetiawan** [comment]
@John-Ragland and @FelixSchwock now the pre-commit works great. In your development environment you should install `pre-commit`. This way, everytime you make changes and try to commit, it will lint the whole code for you before you can commit.

Installing pre-commit: 
1. `$ conda install -c conda-forge pre-commit`
2. `$ pre-commit install`

**Review by John-Ragland — APPROVED** [review]

File: conda_envs/ooi_windows.yml [hunk 1]
```diff
@@ -255,4 +255,4 @@ dependencies:
     - tqdm==4.45.0
     - widgetsnbextension==3.5.1
     - xarray==0.15.1
-prefix: C:\Users\fschwock\AppData\Local\Continuum\anaconda3\envs\obspy
\ No newline at end of file
+prefix: C:\Users\fschwock\AppData\Local\Continuum\anaconda3\envs\obspy
```
Comment (FelixSchwock) [inline conda_envs/ooi_windows.yml:258]: looks like this is pointing to something on my local machine. Do we need this line or does anybody know how we can fix that?

Comment (John-Ragland) [reply conda_envs/ooi_windows.yml:258]: with all of the correct conda / pip installation working, do we need these environment files anymore?

Comment (FelixSchwock) [reply conda_envs/ooi_windows.yml:258]: that's true. We can probably remove those yml files.

Comment (FelixSchwock) [reply conda_envs/ooi_windows.yml:258]: I can remove them. I found another thing that looked a bit awkward after going through the linting.

Comment (FelixSchwock) [reply conda_envs/ooi_windows.yml:258]: Just realized it might be easier to merge this PR first and then do those chnages

Comment (John-Ragland) [reply conda_envs/ooi_windows.yml:258]: sounds good to me.

**Pull request merged**